templates/frontend/pages/indexJournal.tpl

```diff
@@ -26,7 +26,7 @@
 
 	{call_hook name="Templates::Index::journal"}
 
-	{if $showJournalDescription}
+	{if $showJournalSummary && $currentJournal->getLocalizedDescription()}
```

Comment:
Did you mean `$showJournalDescription` variable? e.g.: https://github.com/sssoz/immersion/blob/ui-fixes/ImmersionThemePlugin.inc.php#L292 here `$showJournalDescription` is assigned to Smarty.

Reply:
fixed and force-pushed! oops...